src/PeriodEnd/ManualAdjustments/SFA.DAS.ProviderPayments.Calc.ManualAdjustments/Infrastructure/Dcfs/DcfsPaymentRepository.cs

```diff
@@ -78,7 +78,5 @@ public void CreatePayment(PaymentEntity payment, RequiredPaymentEntity requiredP
             if (payment.FundingSource != 1)
```

Comment:
remove unused usings

Reply:
second line in GetPaymentsForRequiredPayment query does not require $ as there are no values to replace

Reply:
CreateCoInvestmentHistoryPayment method has redundant method parameters which should be removed